README.md

```diff
@@ -3,8 +3,7 @@
 <!-- toc -->
 
 - [Node.js sample app on OpenShift!](#nodejs-sample-app-on-openshift)
-  * [OpenShift Origin v3 setup](#openshift-origin-v3-setup)
-    + [Running a virtual machine with Vagrant](#running-a-virtual-machine-with-vagrant)
+  * [OpenShift v3 setup](#openshift-v3-setup)
```

Comment:
i think we should do away with this section entirely.  There are better docs about how to get a hold of an openshift cluster, we should either point to them, or just leave it as "prereq: you need access to a cluster"

Reply:
I'll update that part to be more inline with the other quickstarts like https://github.com/sclorg/cakephp-ex and https://github.com/sclorg/rails-ex/